src/OrbitGl/MinusButton.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "MinusButton.h"
+
+#include "GlCanvas.h"
+
+namespace orbit_gl {
+
+void MinusButton::DoDraw(PrimitiveAssembler& primitive_assembler, TextRenderer& text_renderer,
+                         const DrawContext& draw_context) {
+  Button::DoDraw(primitive_assembler, text_renderer, draw_context);
```

Comment:
(Personal preference)
If the only difference for `MinusButton` and `PlusButton` is drawing "+" or "-" in `DoDraw`, then we don't need to add these two classes.

We can add a enum class`Button::SymbleType`, and draw different symbols according to the type. The mouse release callback can be passed into the constructor.

Reply:
Nice suggestion. :)